samples/ControlCatalog/App.xaml.cs

```diff
@@ -67,9 +67,9 @@ public class App : Application
 
         public override void Initialize()
         {
-            AvaloniaXamlLoader.Load(this);
-
             Styles.Insert(0, FluentDark);
```

Comment:
FluentDark should be loaded before another resources. Otherwise SideBar will try to read undefined resources.